### ISSUE TITLE
Asset Optimization Minimization and release

### ISSUE THREAD

**Issue by SirMishaa** [busybox11/codesearch#4]
Assets should be optimized with a tool like [**Webpack** ](https://webpack.js.org/)and release should be available.

**Comment by busybox11** [comment]
Yup. I saw an alternative to webpack which claims to be blazing fast compared to webpack and stuff, I'm going to look into that.

All optimizations and improvements are going to be made after I implement a good number of features.
